scout/artifactory.md

```diff
@@ -20,13 +20,25 @@ title: Artifactory integration
 Integrating Docker Scout with JFrog Artifactory lets you run image analysis
 automatically on images in your Artifactory registries.
 
-This integration is made possible by a monitoring agent. The agent is a
+## Local image analysis
+
+You can analyze Artifactory images for vulnerabilities locally using Docker Desktop or the Docker CLI. You first need to authenticate with JFrog Artifactory using the `[Docker login](/engine/reference/commandline/login/)` command. For example:
+
+```bash
+docker login {REPOSITORY_URL}.jfrog.io
```

Comment:
It won't necessarily end in `jfrog.io` for instance if the user has setup their own self-hosted instance.

Reply:
```suggestion
docker login {REPOSITORY_URL}
```

Reply:
I think `REPOSITORY_URL` might not be ideal. Maybe `ARTIFACTORY_URL` or `INSTANCE_URL`?

Reply:
OK sure. I just wasn't sure because Artifactory repository != Docker repository... 😬

Reply:
hehe, yeah